pip_update_no_dash.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Should just change the original to take an option to not update the dashboard

Reply:
changed

Reply:
remove